obspy/core/inventory/response.py

```diff
@@ -1654,6 +1661,9 @@ def get_evalresp_response(self, t_samp, nfft, output="VEL",
         :type end_stage: int, optional
         :param end_stage: Stage sequence number of last stage that will be
             used (disregarding all later stages).
+        :type hide_sensitivity_mismatch_warning: bool
+        :param hide_sensitivity_mismatch_warning: Hide the evalresp warning
+            that computed and reported sensitivities don't match.
```

Comment:
```suggestion
            that computed and reported sensitivities do not match.
```

Reply:
Just a side note: I copied the comment from legacy code so the "don't" is still around